### PR TITLE
Fixes before stable release

### PR DESCRIPTION
**Pull request by podarok**
- remove ability to update plugins. Only install
- remove prompt for variables during jenkinsbox.yml
- move jenkins plugins to cibox-jenkins role defaults
- add install jenkins via debian package
- stick to jenkins version.
- stick to jenkins plugins versions
- remove broken placeholders

Created https://github.com/propeoplemd/cibox/wiki/QUICKSTART

### CODE REVIEW

**Comment by podarok** [comment]
@Sanchiz needs review
I've tested on clean install and looks like now settings for 25 lines still in place after save.
But I would like to ask you to test on your own install, becuase it is large refactoring

File: files/jenkins/jobs/PR_BUILDER/config.xml [hunk 1]
```diff
@@ -50,7 +50,7 @@
     <triggers>
         <org.jenkinsci.plugins.ghprb.GhprbTrigger plugin="ghprb@1.15-0">
             <spec>H/3 * * * *</spec>
-            <adminlist>CHANGE_ME_to_botname
+            <adminlist>
                 podarok
                 ygerasimov
```
Comment (Sanchiz) [inline files/jenkins/jobs/PR_BUILDER/config.xml]: Maybe we can cleanup adminList? Anyway all users from our company team are admins.

File: roles/cibox-jenkins/defaults/main.yml [hunk 2]
```diff
@@ -1,8 +1,30 @@
...
+  - 'http://mirrors.jenkins-ci.org/plugins/ssh-credentials/1.11/ssh-credentials.hpi'
+  - 'http://mirrors.jenkins-ci.org/plugins/hipchat/0.1.9/hipchat.hpi'
+  - 'http://mirrors.jenkins-ci.org/plugins/scm-api/0.2/scm-api.hpi'
+  - 'http://mirrors.jenkins-ci.org/plugins/scripttrigger/0.31/scripttrigger.hpi'
+  - 'http://mirrors.jenkins-ci.org/plugins/parameterized-trigger/2.28/parameterized-trigger.hpi'
+
+jenkins_pkg_version: 'http://pkg.jenkins-ci.org/debian/binary/jenkins_1.624_all.deb'
+
+# Delay before jenkins up and running. Sometimes it is not enouph.
+startup_delay_s: 40
```
Comment (Sanchiz) [inline roles/cibox-jenkins/defaults/main.yml]: Endline

File: roles/cibox-jenkins/tasks/plugins.yml [hunk 3]
```diff
@@ -1,27 +1,9 @@
...
-  sudo: yes
-  action: shell java -jar {{ jenkins.cli_dest }} -s http://localhost:8080 list-plugins | grep ')$' | cut -f 1 -d ' ' | sed ':a;N;$!ba;s/\n/ /g'
-  register: plugins_updates
-
-- name: Update plugins
-  sudo: yes
-  action: command java -jar {{ jenkins.cli_dest }} -s http://localhost:8080 install-plugin {{ plugins_updates.stdout }}
-  when: plugins_updates.stdout != ''
-  notify:
-    - 'Restart Jenkins'
```
Comment (Sanchiz) [inline roles/cibox-jenkins/tasks/plugins.yml]: Endline

**Pull request merged**